model_compression_toolkit/core/pytorch/reader/graph_builders.py

```diff
@@ -318,7 +318,7 @@ def nodes_builder(model: GraphModule,
                 Logger.critical(f'Found FX nodes in framework attributes of {node.name}. This node type should not contain any.')  # pragma: no cover
 
             graph_node_type = BaseNode
-            kwargs = {}
+            kwargs = {INPUTS_AS_LIST: inputs_as_list}
```

Comment:
This shouldn't be the case in a non-functional node.
If this is specific to some custom layer that is not functional but expects input as list then it needs to be handled separately (or modified in custom layers, because it does not follow the protocol of torch modules)